garminconnect/__init__.py

```
@@ -958,7 +958,7 @@ def get_device_last_used(self):
 
         return self.connectapi(url)
 
-    def get_activities(self, start: int = 0, limit: int = 20):
+    def get_activities(self, start: int = 0, limit: int = 20, activitytype = None):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_activities`

(ANN201)